asreview/webapp/src/ProjectComponents/ExportComponents/ExportPage.js

```diff
@@ -147,14 +151,28 @@ const ExportPage = (props) => {
                       onChange={handleFile}
                     >
                       <MenuItem value="dataset" divider>
+                        <Box>
+                          <Typography variant="subtitle1">
+                            Full dataset
+                          </Typography>
+                          <Typography
+                            variant="body2"
+                            gutterBottom
+                            sx={{ color: "text.secondary" }}
+                          >
+                            With relevant and irrelevant labels
```

Comment:
Aren't the unlabelled records not used here? Do you refer to all records in the dataset?